.ci.yaml

```diff
@@ -577,6 +577,9 @@ targets:
     timeout: 60
     dimensions:
       os: "Linux"
+    enabled_branches:
+      # Don't run this on release branches
```

Comment:
I am not sure this is right.

This is definitely used to publish <https://main-api.flutter.dev/>, so that makes sense, but _something_ publishes <https://api.flutter.dev/> (stable branch). Are we sure it wasn't this target?

(If we are sure, let's expand the comment to include the details)

Reply:
What part is not correct?

* release_build: true has always been skipped by release branches
* api.flutter.dev is based on [c519ee916e](https://github.com/flutter/flutter/commit/c519ee916e) which was Jan 21. They must have a process for generating these.
* `release_ref: refs/heads/master` is used by the recipe and is the master branch. I didn't dig too much into `flutter/docs.py`.
* stable and dev loop to be here:

```yaml
  - name: Linux docs_deploy_stable
    recipe: flutter/docs
    scheduler: release
    bringup: true
    enabled_branches:
      - stable
```

```yaml
  - name: Linux docs_deploy_beta
    recipe: flutter/docs
    scheduler: release
    bringup: true
    enabled_branches:
      - beta
```

Reply:
Added clearer notes.